mypyc/irbuild/ll_builder.py

```diff
@@ -762,7 +762,7 @@ def call_c(self,
                 arg = self.coerce(arg, desc.var_arg_type, line)
                 coerced.append(arg)
         target = self.add(CallC(desc.c_function_name, coerced, desc.return_type, desc.steals,
-                                desc.error_kind, line, var_arg_idx))
+                                desc.error_kind, line, var_arg_idx, desc.extra_int_constant))
```

Comment:
As discussed in other comments, it would be better for consistency to create a `LoadInt` for the constant value and append it to `coerced`.